packages/nestjs-zod/src/dto.test.ts

```diff
@@ -72,6 +72,35 @@ describe('zod/v4', () => {
       myField: expect.objectContaining({ type: 'string', required: true, default: 'myField' })
     })
   })
+
+  it('handles z.date() fields in output schemas with format: date-time', () => {
+    const UserSchema = z4.object({
+      name: z4.string(),
+      birthDate: z4.date(),
+      createdAt: z4.date().optional(),
+    })
+  
+    class UserDto extends createZodDto(UserSchema) {}
+
+    // Should not crash when generating OpenAPI metadata
+    const metadata = UserDto.Output._OPENAPI_METADATA_FACTORY();
+    
+    expect(metadata).toEqual({
+      name: expect.objectContaining({ type: 'string', required: true }),
+      birthDate: expect.objectContaining({ 
+        type: 'string', 
+        format: 'date-time', 
+        required: true 
+      }),
+      createdAt: expect.objectContaining({ 
+        type: 'string', 
+        format: 'date-time', 
+        required: false 
+      })
+    })
+  })
+
+
 })
```

Comment:
Add back the test that assets an error is thrown for input schemas.  You don't need to add an explicit throw.  Zod will throw.  Just assert zod throws